test/scss/fixtures/bitstyles-overrides.css

```diff
@@ -599,12 +599,12 @@ h3,
 h4,
 h5,
 h6 {
-  text-rendering: optimizeLegibility;
   font-family: FakeFont, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
-    helvetica, arial, sans-serif, Apple Color Emoji, Segoe UI Emoji,
-    Segoe UI Symbol;
+  helvetica, arial, sans-serif, Apple Color Emoji, Segoe UI Emoji,
+  Segoe UI Symbol;
```

Comment:
Does this come from the linter/formatter? Makes it harder to read in my opinion, but if they think it's a good idea - fine.